server/server/models/user.py

```diff
@@ -109,16 +92,81 @@ def __init__(self, email, nickname, password, create_date, location, first_name,
         self.avatar = avatar
         self.banned_to_date = banned_to_date
 
+
+def from_json(cls, data):
+    """ Deserialize User object from JSON """
+    return cls(**data)
+
+
+def model_to_json(sqlalchemy_object):
+    fields_arr = [prop.key for prop in
+                  class_mapper(sqlalchemy_object.__class__).iterate_properties
+                  if isinstance(prop, ColumnProperty)]
+    _dict = {}
+    for key in fields_arr:
+        _dict[key] = str(getattr(sqlalchemy_object, key))
+    return _dict
+
+
 def users_to_json(obj):
     """ Serialize User object to JSON """
-
-    to_serialize = ['id', 'email', 'nickname', 'password', 'create_date', 'location', 'first_name', 'last_name', 'status_id', 'role_id', 'avatar', 'banned_to_date']
+    to_serialize = ['id', 'email', 'nickname', 'password', 'create_date',
+                    'location', 'first_name', 'last_name', 'status_id',
+                    'role_id', 'avatar', 'banned_to_date']
     d = {}
     for attr_name in to_serialize:
-        d[attr_name] = getattr(obj, attr_name)
+        d[attr_name] = str(getattr(obj, attr_name))
     return d
 
-def from_json(cls, data):
-    """ Deserialize User object from JSON """
 
-    return cls(**data)
+def get_all_users():
+    """ Method to get all users from database """
+    qry = get_dbsession().query(User).all()
+    usr_in_json = list(map(lambda x: model_to_json(x), qry))
+    return usr_in_json
+
+
+def create_usr(json_str):
+    """ Method to add user to database """
+    user = User(**json_str)
+    get_dbsession().add(user)
+    transaction.commit()
+    return {'status': 'Success!'}
+
+
+def read_usr(request):
+    """ Method to read user from database """
+    try:
+        qry = get_dbsession().query(User)\
+            .filter(User.id == request.matchdict['user_id']).first()
+        dict_ = users_to_json(qry)
+    except:
+        request.response.status = 404
+        return {'status': '404'}
+    return json.loads(json.dumps(dict_))
+
+
+def update_usr(json_str, request):
+    """ Method to update user in database """
+    json_encoded = json.dumps(json_str)
+    try:
+        get_dbsession().query(User)\
+            .filter(User.id == request.matchdict['user_id']).\
+            update(json.loads(json_encoded))
+        transaction.commit()
+    except:
+        request.response.status = 404
+        return {'status': '404'}
```

Comment:
If you want to return status 404, better to use pyramid.httpexceptions
[https://docs.pylonsproject.org/projects/pyramid/en/latest/api/httpexceptions.html](url)